### PR TITLE
zayisula w

### PR DESCRIPTION
**Pull request by jkunimune15**
sa karibi mare, pan **w** mutu pa **v** sa logoliste. a multi **w** restu sa ali kitabe, misalo baxkanune. ye tirplize xula pan vi **w**, e nova ali purani loge sa kitabe.

### CODE REVIEW

File: engli/baze.md [hunk 1]
```diff
@@ -27,8 +27,7 @@ This document is a simple guide to the main features of Pandunia.
 | s | /s/ |
 | t | /tʰ/ |
 | u | /u/ |
-| v | /v/ |
-| w | /w~ʋ/ |
+| w | /w~v/ |
```
Comment (barumau) [inline engli/baze.md:30]: Should be:
| v | /w~ʋ~v/ |

The labiodental approximant /ʋ/ is a likely realization of "v" in certain accents.

File: engli/dunia_baxe.md [hunk 2]
```diff
@@ -387,13 +387,13 @@ form that sounds familiar to speakers of as many languages as possible.
...
 | **kort**      | short      | curto      | corto      | court      | kurz       | korotkiy   |
-| **now**       | new        | novo       | nuevo      | nouveau    | neu        | novıy      |
+| **nov**       | new        | novo       | nuevo      | nouveau    | neu        | novıy      |
 | **marc**      | march      | marcha     | marcha     | marche     | Marsch     | marš       |
 | **post**      | post (mail)| (postal)   | posta      | poste      | Post       | počta      |
 
 The suffixes of Pandunia are applied on the roots. Here are some
 resulting Pandunia words: korti (short), korte (shorty), korta 
-(shorten), nowi (new), nowe (news), nowa (renovate).
+(shorten), nov (new), nove (news), nova (renovate).
```
Comment (barumau) [inline engli/dunia_baxe.md:396]: novi. Adjectives normally include -i and I don't see why "nov" would be an exception anymore.

File: engli/baze.md (hunk 1 above)
Comment (jkunimune15) [reply engli/baze.md]: I just fixed it.

File: engli/dunia_baxe.md (hunk 2 above)
Comment (jkunimune15) [reply engli/dunia_baxe.md]: Good catch!

**Pull request merged**